prow/github/client.go

```diff
@@ -1426,3 +1426,22 @@ func (c *Client) IsMergeable(org, repo string, number int, sha string) (bool, er
 	}
 	return false, fmt.Errorf("reached maximum number of retries (%d) checking mergeability", maxTries)
 }
+
+// ClearMilestone clears the milestone from the specified issue
+func (c *Client) ClearMilestone(org, repo string, num int) error {
+	c.log("ClearMilestone", org, repo, num)
+	if c.fake {
+		return nil
+	}
+
+	issue := &struct {
+		Milestone interface{} `json:"milestone"`
```

Comment:
Is this `interface{}` just because you want the serialized value to be `null` and don't need a type?

Reply:
Correct.  afaik this is the only way to clear the milestone.

Reply:
Please add a short comment explaining this.

Reply:
Done.